esmvaltool/recipes/recipe_quantilebias.yml

```diff
@@ -0,0 +1,86 @@
+# recipe_quantilebias.yml
+---
+
+documentation:
+  description: |
+    Tool for calculation of precipitation quantile bias
+
+  authors:
+    - arno_en
+    - hard_jo
+
+  references:
+    - mehran14jgr
+
+  projects:
+    - c3s-magic
+
+datasets:
+  - {dataset: EC-EARTH, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1997, end_year: 1997}
+ # - {dataset: HadCM3, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1997, end_year: 1997}
+ # - {dataset: MPI-ESM-MR, project: CMIP5, exp: historical, ensemble: r1i1p1, start_year: 1997, end_year: 1997}
+  - {dataset: GPCP-SG, project: obs4mips, level: L3, version: v2.3, tier: 1, start_year: 1997, end_year: 1997}
```

Comment:
On Jasmin we have `version: v2.2` available, can that also be used? If not, I will have to ask Bjoern to update.

Reply:
I guess that also v2.2 should be ok, what do you think @earnone ?